scripts/build_web.py

```diff
@@ -0,0 +1,37 @@
+from shutil import copyfile
+from subprocess import check_output
+import sys
+import os
+import platform
+import subprocess
+
+Import("env")
+
+
+def is_tool(name):
+    cmd = "where" if platform.system() == "Windows" else "which"
+    try:
+        check_output([cmd, name])
+        return True
+    except:
+        return False;
+
+
+def build_web():
+    if is_tool("npm"):
+        os.chdir("web")
+        print("Attempting to build webpage...")
+        try:
+            print check_output(["npm", "install"])
```

Comment:
This is not python 3 compatible. Please can you make it python 3 compatible? It is worse enough that PlatformIO still not run with python 3.